.github/workflows/build.yml

```
@@ -25,7 +25,7 @@ jobs:
           sudo apt-get update && sudo apt-get install -y osslsigncode
           cp LICENSE "$RUNNER_TEMP/LICENSE"
           echo -e "\n---\n" >> "$RUNNER_TEMP/LICENSE"
-          curl "https://golang.org/LICENSE?m=text" >> "$RUNNER_TEMP/LICENSE"
+          curl "https://go.dev/LICENSE?m=text" >> "$RUNNER_TEMP/LICENSE"
```

Let's also follow redirects to be a little more forward compatible.
```suggestion
          curl -L "https://go.dev/LICENSE?m=text" >> "$RUNNER_TEMP/LICENSE"
```